test/csuite/timestamp_abort/main.c

```diff
@@ -773,6 +773,13 @@ main(int argc, char *argv[])
                 if (c_rep[i].first_miss == INVALID_KEY)
                     c_rep[i].first_miss = key;
                 c_rep[i].absent_key = key;
+            } else if ((ret = cur_shadow->search(cur_shadow)) != 0) {
+                if (ret != WT_NOTFOUND)
+                    testutil_die(ret, "shadow search");
+                else {
+                    printf("%s: SHADOW no record with key %" PRIu64 "\n", fname, key);
+                    absent_shadow++;
```

Comment:
@ddanderson although this has been merged, I suggest possibly opening a new ticket. There are two halves to the collection/shadow table testing and interestingly, your change did not modify the lines at either of those two locations. So it may be that you want to just increment and print in those two location instead of adding in a 3rd location.

The first consistency check detection is at the line that says `testutil_die(ret, "shadow search success")`. In that case, we did not find a record in the collection, but *did* find that same record in the shadow table. The second consistency check is at the line that says "testutil_die(ret, "shadow search failure")`. In that case we did find the record in the collection, but did not find that record in the shadow table.